src/status_im/ui/screens/chat/message/message_old.cljs

```diff
@@ -608,27 +661,37 @@
                                    :id       :delete}]))))}]
    reaction-picker])
 
-(defmethod ->message constants/content-type-audio []
+(defmethod ->message constants/content-type-audio
+  []
   (let [show-timestamp? (reagent/atom false)]
-    (fn [{:keys [outgoing] :as message}
-         {:keys [on-long-press modal]
-          :as   reaction-picker}]
+    (fn
+      [{:keys [outgoing] :as message}
+       {:keys [on-long-press modal]
+        :as   reaction-picker}]
       [message-content-wrapper message
        [react/touchable-highlight
         (when-not modal
           {:on-long-press
-           (fn [] (on-long-press (if (and outgoing config/delete-message-enabled?)
-                                   [{:on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
-                                     :label    (i18n/label :t/delete)
-                                     :id       :delete}]
-                                   [])))
+           (fn []
+             (on-long-press
+              (concat
+               [{:on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
+                 :id       :reply
+                 :label    (i18n/label :t/message-reply)}]
+               (if (and outgoing config/delete-message-enabled?)
+                 [{:on-press #(re-frame/dispatch [:chat.ui/soft-delete-message message])
+                   :label    (i18n/label :t/delete)
+                   :id       :delete}]
+                 []))))
            :on-press (fn []
                        (reset! show-timestamp? true))})
         [react/view (style/message-view-wrapper (:outgoing message))
          [message-timestamp message show-timestamp?]
          [react/view {:style (style/message-view message) :accessibility-label :audio-message}
           [react/view {:style (style/message-view-content)}
-           [message.audio/message-content message] [message-status message]]]]]
+           [react/view {:style (style/style-message-text outgoing)}
```

Comment:
why extra view ? its better to have less views as possible

Reply:
That's not my code. This is showing as a change due to formatting changes by the IDE. To fix all the unnecessary formatting changes caused by the IDE, I opened a new [PR](https://github.com/status-im/status-mobile/pull/13808). Please follow on the new [PR](https://github.com/status-im/status-mobile/pull/13808).